### DISCUSSION TITLE
Customisable (Weekend) Auto Freeze Streaks

### DISCUSSION THREAD

**Discussion by FahimSakib** [dailydotdev/daily#1471] [Feature Requests]
It would be great if I can customise the the weekend day for the streak freeze. For example In BD we have weekend on various format like, Friday - Saturday; Thursday - Friday; Saturday - Sunday; etc.

**Comment by idoshamun** [comment]
Hi @FahimSakib,

Currently we support the most common weekends, fri-sat and sat-sun. You can manage it here: https://app.daily.dev/account/notifications.

**Reply by FahimSakib** [reply]
Yes, that's why I requested a customisable weekend, which would be great if you could add it (though it does not matter much).

Thanks for the reply @idoshamun

**Comment by Nyakuvengwa** [comment]
I recently lost my daily.dev streak due to a public holiday and believe that a feature to automatically pause streaks for country-specific holidays would be very helpful.

While the idea of using reputation points to restore a streak is a good start, a proactive pausing feature would be a great quality-of-life improvement.

**Reply by idoshamun** [reply]
I understand that losing your streak can be frustrating, hence we introduced the streak restore which now costs Cores, not reputation. Would you want to be able to proactively freeze your streak for Cores?
